tabs/src/components/teamsfx/Tab.jsx

```diff
@@ -1,11 +1,37 @@
 import React from "react";
+import { Button, Header, Text } from '@fluentui/react-northstar'
+import Deeplink from './tab/Deeplink.jsx'
+import StaticTab from './tab/StaticTab.jsx'
+import { TopJumper, ScrollToAnchor } from "../util/ScrollUtil";
+import Code from "../util/CodeUtil"
 
-class Tab extends React.Component {
-  render() {
-    return <div>
-      <h1>Tab</h1>
+function InitializeCode() {
+  const initializeDefinition = '// @microsoft.teams-js\n// import * as microsoftTeams from "@microsoft/teams-js";\nfunction initialize(callback?: () => void, validMessageOrigins?: string[])'
```

Comment:
Better to put hard coded code to resource file, so that it easy to maintain

Reply:
ok, I will put it in a new folder under '/src/code/tab'